### PR TITLE
sys-kernel/pf-sources: add 6.6_p3

### PR DESCRIPTION
**Pull request by dagraver**
Closes: https://bugs.gentoo.org/910186

### CODE REVIEW

File: sys-kernel/pf-sources/pf-sources-6.6_p3.ebuild [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+ETYPE="sources"
+
+# Use genpatches but don't include the 'experimental' use flag.
+K_EXP_GENPATCHES_NOUSE="1"
+
+# Genpatches version to use. -pf patch set already includes vanilla linux updates. Regularly "1"
+# is the wanted value here, but the genpatches patch set can be bumped if it includes some
+# important fixes. src_prepare() will handle deleting the updated vanilla linux patches.
+# See https://archives.gentoo.org/gentoo-kernel/ (or subscribe to the list) to see all patches.
+K_GENPATCHES_VER="1"
```
Comment (juippis) [inline sys-kernel/pf-sources/pf-sources-6.6_p3.ebuild:16]: Should we bump this to latest? I haven't kept a close eye on kernel development anymore, but still I've heard about breaking issues in 6.6. IIRC our genpatches have been bumped to fix some of these at least. 

Then there's the ext4 corruption bug, is that fixed in this release? We should bump to a version that addresses the ext4 issue.

Comment (juippis) [reply sys-kernel/pf-sources/pf-sources-6.6_p3.ebuild:16]: @zkl-dev let's hear your comments too, if you have an opininon / more information on this.

Comment (dagraver) [reply sys-kernel/pf-sources/pf-sources-6.6_p3.ebuild]: You are, of course, correct. An oversight on my part. I've bumped it to 7, as is used in the 'official' gentoo-sources-6.6.5.

**Review by juippis — APPROVED** [review]

File: sys-kernel/pf-sources/pf-sources-6.6_p3.ebuild (hunk 1 above)
Comment (zkl-dev) [reply sys-kernel/pf-sources/pf-sources-6.6_p3.ebuild]: > @zkl-dev let's hear your comments too, if you have an opininon / more information on this.

Hi, I did the same "mistake" / "oversight" in my update as well :).

I think matching the gentoo-source-6.6.5 was the right thing to do.

Thanks for keeping me in the loop, I appreciate the opportunity to learn ! :)

**Pull request closed without merging**